features/steps/ios_steps.rb

```diff
@@ -16,10 +16,15 @@
 end
 
 When("I run {string} and relaunch the app") do |event_type|
-  steps %Q{
-    When I run "#{event_type}"
-    And I relaunch the app after a crash
-  }
+  begin
+    step("I run \"#{event_type}\"")
+  rescue StandardError
+    # AppiumForMac raises an to run a scenario that crashes the app
```

Comment:
I'm finding it difficult to understand this comment 😅

Reply:
Seems picky 😂 , but reworded.